android/src/main/java/com/amplitude/android/Amplitude.kt

```diff
@@ -27,13 +27,11 @@ open class Amplitude(
     private lateinit var androidContextPlugin: AndroidContextPlugin
 
     override fun build() {
-        val storageDirectory = (configuration as Configuration).context.getDir("${FileStorage.STORAGE_PREFIX}-${configuration.instanceName}", Context.MODE_PRIVATE)
         idContainer = IdentityContainer.getInstance(
             IdentityConfiguration(
                 instanceName = configuration.instanceName,
                 apiKey = configuration.apiKey,
-                identityStorageProvider = FileIdentityStorageProvider(),
-                storageDirectory = storageDirectory
```

Comment:
would the identity storage still create correctly in this case? The identity module will trigger the initialization right?

Reply:
moved the storage init after set the directory path